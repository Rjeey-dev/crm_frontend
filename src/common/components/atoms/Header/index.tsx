import {ReactNodeArray} from "prop-types";
import React, {ReactNode} from "react";

interface IProps {
    classes?: string,
    id?: string,
    children?: ReactNodeArray | ReactNode
}

function Header(props: IProps) {
    return <header className={props.classes} id={props.id}>{props.children}</header>;
}

export default Header;