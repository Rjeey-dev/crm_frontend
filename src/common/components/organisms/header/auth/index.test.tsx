import {shallow} from "enzyme";
import React from 'react';

import Header from './index';

jest.mock('containers/blocks/Notifications', () => 'notifications');
jest.mock('containers/blocks/Settings/OpenCloseButton', () => 'open-close-button');
jest.mock('containers/links/LogOut', () => 'logout-link');

it("Renders <Header/> with correct properties", () => {
    const output = shallow(
        <Header/>
    ).html();

    expect(output).toMatchSnapshot();
});