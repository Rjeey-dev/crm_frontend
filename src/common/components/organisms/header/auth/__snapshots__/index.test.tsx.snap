// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Header/> with correct properties 1`] = `"<nav class=\\"navbar default-layout col-lg-12 col-12 p-0 fixed-top d-flex flex-row\\"><div class=\\"text-center navbar-brand-wrapper d-flex align-items-top justify-content-center\\"><a href=\\"dashboard\\" class=\\"navbar-brand brand-logo\\"><img src=\\"images/logo.svg\\" alt=\\"Logo\\"/></a><a href=\\"dashboard\\" class=\\"navbar-brand brand-logo-mini\\"><img src=\\"images/logo-mini.svg\\" alt=\\"Logo\\"/></a></div><div class=\\"navbar-menu-wrapper d-flex align-items-center\\"><ul class=\\"navbar-nav navbar-nav-left header-links d-none d-md-flex\\"><li class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\"><span class=\\"badge badge-primary ml-1\\"></span>Missing translationId: menus.header_links.shedule for language: \${ languageCode }</a></li><li class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\"><i class=\\"mdi mdi-elevation-rise\\"></i>Missing translationId: menus.header_links.reports for language: \${ languageCode }</a></li><li class=\\"nav-item\\"><a href=\\"#\\" class=\\"nav-link\\"><i class=\\"mdi mdi-bookmark-plus-outline\\"></i>Missing translationId: menus.header_links.score for language: \${ languageCode }</a></li></ul><ul class=\\"navbar-nav navbar-nav-right\\"><li class=\\"nav-item dropdown\\"><notifications></notifications></li><li class=\\"nav-item dropdown\\"><open-close-button></open-close-button></li><li class=\\"nav-item dropdown d-none d-xl-inline-block\\"><div><a href=\\"#\\" id=\\"UserDropdown\\" class=\\"nav-link dropdown-toggle\\" aria-expanded=\\"false\\" data-toggle=\\"dropdown\\"><span class=\\"profile-text\\">Missing translationId: common.hello_with_name for language: \${ languageCode }</span><img class=\\"img-xs rounded-circle\\" src=\\"images/faces/face1.jpg\\" alt=\\"Profile image\\"/></a><div class=\\"dropdown-menu dropdown-menu-right navbar-dropdown\\"><div class=\\"d-flex border-bottom\\"><div class=\\"py-3 px-4 d-flex align-items-center justify-content-center\\"><a href=\\"#\\"><i class=\\"mdi mdi-bookmark-plus-outline mr-0 text-gray\\"></i></a></div><div class=\\"py-3 px-4 d-flex align-items-center justify-content-center border-left border-right\\"><a href=\\"#\\"><i class=\\"mdi mdi-account-outline mr-0 text-gray\\"></i></a></div><div class=\\"py-3 px-4 d-flex align-items-center justify-content-center\\"><a href=\\"#\\"><i class=\\"mdi mdi-alarm-check mr-0 text-gray\\"></i></a></div></div><a href=\\"#/profile\\" class=\\"dropdown-item mt-2\\">Missing translationId: menus.header_user_menu.manage_account for language: \${ languageCode }</a><a href=\\"#\\" class=\\"dropdown-item\\">Missing translationId: menus.header_user_menu.change_password for language: \${ languageCode }</a><a href=\\"#\\" class=\\"dropdown-item\\">Missing translationId: menus.header_user_menu.check_inbox for language: \${ languageCode }</a><logout-link></logout-link></div></div></li></ul></div></nav>"`;
